IntelliTect.TestTools.TestFramework/ExampleTests/ExampleTests/WebDriverFactory.cs

```diff
@@ -17,14 +17,17 @@ public WebDriverFactory(string browserType)
 
         private IWebDriver GetWebDriver(IServiceProvider service)
         {
-            if (_BrowserType == "Chrome")
-                _Driver = new ChromeDriver(Directory.GetCurrentDirectory());
+            IWebDriver? driver;
+            if (_BrowserType is "Chrome")
+                driver = new ChromeDriver(Directory.GetCurrentDirectory());
+            // Add other browser here.
             else
-                _Driver = new ChromeDriver(Directory.GetCurrentDirectory());
-            return _Driver;
+                driver = new ChromeDriver(Directory.GetCurrentDirectory());
```

Comment:
Do we actually need this if statement? it looks like both sides do the same thing.

Reply:
I was going to add other drivers to show one way of handling cross-browser testing, buuuuut... I think I'm going to skip that. Removed.